src/main/java/com/launchcode/AroundTownServer/models/User.java

```diff
@@ -0,0 +1,81 @@
+package com.launchcode.AroundTownServer.models;
+
+import android.support.annotation.Size;
+import com.sun.istack.NotNull;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+
+import javax.persistence.Table;
+import javax.swing.*;
+
+@Entity
+@Table(name = "user")
```

Comment:
unsure if this table annotation is needed, may not be necessary

Reply:
it just names the table, so it shouldn't affect anything else